uswds/uswds-sandbox
# USWDS-Sandbox - Compile: Test 1.2.0 w/ USWDS v2

# Description

Testing USWDS compile develop branch with USWDS v2 on sandbox.

> [!WARNING]
> Not intended for merge into `develop`.  Just set merge target to display changes correctly.

## Testing instructions

Run through various gulp scripts using `npx gulp ...` and confirm they run as expected

**Note:** There is a problem building federalist preview. This is unrelated to the compile change and is happening on the `develop` branch already. Only use this branch to test compile features.
